docs/networking/multus-ipams.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Multus and IPAM plugins"
+weight: 25
+---
+
+[Multus CNI](https://github.com/k8snetworkplumbingwg/multus-cni) is a CNI plugin that enables attaching multiple network interfaces to pods. Multus does not replace CNI plugins, instead it acts as a CNI plugin multiplexer. Multus is useful in certain use cases, especially when pods are network intensive and require extra network interfaces that support dataplane acceleration techniques such as SR-IOV.
+
+Multus can not be deployed standalone. It always requires at least one conventional CNI plugin that fulfills the Kubernetes cluster network requirements. That CNI plugin becomes the default for Multus, and will be used to provide the primary interface for all pods. When deploying K3s with default options, that CNI plugin is Flannel.
+
+### Add the Helm Repository
+
+To deploy Multus, we recommend using the following helm repo:
+```
+helm repo add rke2-charts https://rke2-charts.rancher.io
+helm repo update
+```
+
+### Configure the IPAM plugin
+
+An IPAM plugin is required to assign IP addresses on the extra interfaces created by Multus.
+
+<Tabs groupId = "MultusIPAMplugins">
+<TabItem value="host-local" default>
+The host-local IPAM plugin allocates ip addresses out of a set of address ranges. It stores the state locally on the host filesystem, hence ensuring uniqueness of IP addresses on a single host. Therefore, we don't recommend it for multi-node clusters. This IPAM plugin does not require any extra deployment. For more information: https://www.cni.dev/plugins/current/ipam/host-local/.
+
+To use the host-local plugin, please create a file called `multus-values.yaml` with the following content:
```

Comment:
In which folder should the user create this file?

Reply:
It does not matter, it is used later as values for the helm install command: 
```
helm install multus rke2-charts/rke2-multus -n kube-system --kubeconfig /etc/rancher/k3s/k3s.yaml --values multus-values.yaml
```

Reply:
Ok I was confused. I thought it was used to deploy multus automatically